src/functions/length.ts

```diff
@@ -10,31 +12,34 @@ export const length: IFunction<ILengthRuleOptions> = (targetVal, opts) => {
 
   const { min, max } = opts;
 
-  let value;
-  const valueType = typeof targetVal;
-  if (valueType === 'object') {
+  let value: number;
+  if (isPlainObject(targetVal)) {
     value = Object.keys(targetVal).length;
   } else if (Array.isArray(targetVal)) {
-    value = targetVal.length + 1;
-  } else if (valueType === 'number') {
+    value = targetVal.length;
```

Comment:
so adding `+ 1` was a bug? Just want to double check since this was not specifically mentioned in the issue

Reply:
Yeah, was a bug.